### PR TITLE
[Enhance] #27 Add opengraph meta

### PR DESCRIPTION
**Pull request by anxolerd**
[ https://github.com/GDG-Ukraine/gdg.org.ua/issues/27 ](https://github.com/GDG-Ukraine/gdg.org.ua/issues/27)

Add opengraph meta tags to
- Event page
- Registration form page

### CODE REVIEW

**Comment by anxolerd** [comment]
Review request: /cc @webknjaz 

P.S. Do we have any staging resource, we can use for testing purposes?

File: src/GDGUkraine/templates/event.html [hunk 1]
```diff
@@ -9,6 +9,19 @@
 	<meta name="author" content="Olexandr Kovalchuk">
 	<meta name="author" content="Svyatoslav Sydorenko">
 	<meta name="author" content="Valentyn Shybanov">
+    <meta name="author" content="{{ event.host_gdg.name or 'GDG Ukraine' if event.host_gdg_id else 'GDG Ukraine'}}"/>
```
Comment (webknjaz) [inline src/GDGUkraine/templates/event.html]: Plz mind indentation.

Comment (webknjaz) [inline src/GDGUkraine/templates/event.html]: Does ternary operator exist in Jinja2?

File: src/GDGUkraine/templates/register.html [hunk 2]
```diff
@@ -379,7 +392,7 @@
     </div>
     <div id="footer" style="text-align: center; color: rgba(50, 50, 50, 0.7);">
         <small>
-            Regform by <a href="https://google.com/+OlexandrKovalchuk">Olexandr Kovalchuk</a>, <a href="https://plus.google.com/102989169039151799851/about">Oleh Zasadnyy</a>, <a href="https://plus.google.com/112452406409308053659/about">Valentyn Shybanov</a> and <a href="https://plus.google.com/110108248841907526802/about">Svyatoslav Sydorenko</a> | <a href="https://github.com/GDG-Ukraine/gdg.org.ua/issues">Report an issue</a><br />
+            Regform by <a href="https://plus.google.com/102989169039151799851/about">Oleh Zasadnyy</a>, <a href="https://plus.google.com/112452406409308053659/about">Valentyn Shybanov</a>, <a href="https://plus.google.com/110108248841907526802/about">Svyatoslav Sydorenko</a> and <a href="https://google.com/+OlexandrKovalchuk">Oleksandr Kovalchuk</a> | <a href="https://github.com/GDG-Ukraine/gdg.org.ua/issues">Report an issue</a><br />
```
Comment (webknjaz) [inline src/GDGUkraine/templates/register.html]: 1. Why did you do this?
1. How does it relate to the feature?

Comment (anxolerd) [inline src/GDGUkraine/templates/register.html]: 1. I do not like my name stays first in a list like I'm the main developer, which is not true
2. It doesn't. I thought it would be handy to do it now than later.

File: src/GDGUkraine/templates/event.html (hunk 1 above)
Comment (anxolerd) [inline src/GDGUkraine/templates/event.html]: Yes, of course it does)
http://jinja.pocoo.org/docs/dev/templates/#if-expression

File: src/GDGUkraine/templates/event.html [hunk 3]
```diff
@@ -1,24 +1,37 @@
 <!DOCTYPE html>
 <html>
 <head>
-	<meta charset="UTF-8">
-	<base href="/"></base>
+    <meta charset="UTF-8">
+    <base href="/"></base>
```
Comment (webknjaz) [inline src/GDGUkraine/templates/event.html]: Changing indentation style MUST be a separate PR, since it doesn't relate to feature. Also this makes code review more complicated. Please use same indentation style as before and move refactoring to separate PR.

**Comment by anxolerd** [comment]
@webknjaz so if I got that right I have to:

1. Close this PR
2. Fix indentation in separate PR
3. Redo this PR with related functional only
4. If I want to change position of my name on page I should do it in `master` without creating a pull request as it is not related to any functionality.

Did I understand you correctly?

**Comment by webknjaz** [comment]
It would be easier to use old indentation in this PR first and move other changes out of it.
Also you may merge any changes which don't need review.

P.S. There's dev.gdg.org.ua running as a staging.

**Comment by anxolerd** [comment]
Closing as it did not pass the review.

Resolution:
- Move indentation fixes to master
- Create a new pure PR with adding opengraph meta ony.

**Comment by webknjaz** [comment]
You might just merge changes from `master`. This would have produced clean PR.
